test/built-ins/TypedArrayConstructors/ctors/object-arg/returns.js

```diff
@@ -35,7 +35,7 @@ testWithTypedArrayConstructors(function(TA) {
   assert.sameValue(typedArray.constructor, TA);
   assert.sameValue(Object.getPrototypeOf(typedArray), TA.prototype);
 
-  if (TA === Float32Array || TA === Float64Array) {
+  if (TA === Float32Array || TA === Float64Array || (typeof Float16Array !== 'undefined' && TA === Float16Array)) {
```

Comment:
I've seen this condition a bit too often in this PR; let's add a helper function to `testTypedArray.js` (or would we prefer `floatArrayConstructors.indexOf...`?)